react/features/filmstrip/components/native/Filmstrip.js

```diff
@@ -56,6 +70,12 @@ class Filmstrip extends Component<Props> {
      */
     _separateLocalThumbnail: boolean;
 
+    /**
+     * The config to tell the {@code FlatList} component when to show/hide
+     * clipped or not visible children.
+     */
+    _viewabilityConfig: Object;
```

Comment:
why is this a prop if it's then "hardcoded" below?

Reply:
It's not a prop, it's a field in the class

Reply:
:man_facepalming: